packages/ui/theme/foundations/fonts.ts

```diff
@@ -1,8 +1,8 @@
 import type { ThemeOverride } from '@chakra-ui/react';
-import primitives from '@core-ds/primitives';
+// import primitives from '@core-ds/primitives';
```

Comment:
Would it make more sense to change `primitives` rather than diverging in this repo?

Reply:
Definitely. That's precisely the reason why I've opened this PR only as a draft.
In this way we can discuss if the new font set is ok and if so proceed changing `primitives`

Reply:
I opened a PR! https://github.com/iFixit/core-primitives/pull/36